conda/gateways/disk/create.py

```diff
@@ -253,9 +253,7 @@ def make_menu(prefix, file_path, remove=False):
     Passes all menu config files %PREFIX%/Menu/*.json to ``menuinst.install``.
     ``remove=True`` will remove the menu items.
     """
-    if not on_win:
-        return
-    elif basename(prefix).startswith("_"):
+    if basename(prefix).startswith("_"):
         log.warn(
             "Environment name starts with underscore '_'. Skipping menu installation."
         )
```

Comment:
This looks like another remnant of the private environments and should simply be removed (xref https://github.com/conda/conda/pull/12101).

Reply:
conda-build environments (`_h_env....`, `_build_env`, `_test_env...`) are also underscore-envs, though, and maybe this was part of it. However, since menuinst exceptions are caught and logged but not raised, I think it's safe to remove.